builtin/logical/database/backend.go

```diff
@@ -57,6 +60,20 @@ func Factory(ctx context.Context, conf *logical.BackendConfig) (logical.Backend,
 	b.credRotationQueue = queue.New()
 	// Load queue and kickoff new periodic ticker
 	go b.initQueue(b.queueCtx, conf, conf.System.ReplicationState())
+
+	// collect metrics on number of plugin instances
+	pluginInstanceGaugeProcess, err := metricsutil.NewGaugeCollectionProcess(
```

Comment:
Looks like you're missing an assignment to `b.gaugeCollectionProcess` here?

Reply:
Oops! Good catch :)